interfaces/webidl.idl

```diff
@@ -9,7 +9,7 @@ typedef (Int8Array or Int16Array or Int32Array or
          Float32Array or Float64Array or DataView) ArrayBufferView;
 
 typedef (ArrayBufferView or ArrayBuffer) BufferSource;
-[Exposed=*,
+[Exposed=(Window,Worker),
```

Comment:
This reverts https://github.com/web-platform-tests/wpt/pull/36133. Shouldn't do that.